--- hardware/arduino/avr/cores/arduino/USBCore.cpp	(revision )
+++ hardware/arduino/avr/cores/arduino/USBCore.cpp	(revision )
@@ -22,6 +22,8 @@
 
 #if defined(USBCON)
 
+//#define CDC_ENABLED
+
 /** Pulse generation counters to keep track of the number of milliseconds remaining for each pulse type */
 #define TX_RX_LED_PULSE_MS 100
 volatile u8 TxLEDPulse; /**< Milliseconds remaining for data Tx LED pulse */
@@ -359,8 +361,10 @@
 {
 	u8 i = setup.wIndex;
 
+#ifdef CDC_ENABLED
 	if (CDC_ACM_INTERFACE == i)
-		return CDC_Setup(setup);
+		return CDC_Setup(setup);
+#endif
 
 #ifdef PLUGGABLE_USB_ENABLED
 	return PluggableUSB().setup(setup);
@@ -439,7 +443,9 @@
 {
 	u8 interfaces = 0;
 
-	CDC_GetInterface(&interfaces);
+#ifdef CDC_ENABLED
+	CDC_GetInterface(&interfaces);
+#endif
 
 #ifdef PLUGGABLE_USB_ENABLED
 	PluggableUSB().getInterface(&interfaces);
